src/com/company/Main.java

```diff
@@ -3,6 +3,106 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+	// Задача 1
+        byte apple = 4;
+        short pear = 5;
+        int banana = 10;
+        long cherry = 15;
+        float cherryWeight = 1.5f;
+        double bananaWeight = 3.2;
+        boolean cherryWeightIsEnough = cherryWeight > 2;
+        char bar = 35;
+
+    // Задача 2
+    double fighter1 = 78.2;
+    double fighter2 = 82.7;
+    double fightersTotalWeight = fighter1 + fighter2;
+    double fightersWeightDifference = fighter2 - fighter1;
+    System.out.println("Общий вес " + fightersTotalWeight);
+    System.out.println("Разница в весе " + fightersWeightDifference);
+
+    // Задача 3
+        int banana1 = 80;
+        int milk = 105;
+        int icecream = 100;
+        int egg = 70;
+        int babana1TotalWeight = banana1 * 5;
+        System.out.println(babana1TotalWeight + " грамм");
+        int milkTotalWeight = milk * 2;
+        System.out.println(milkTotalWeight + " грамм");
+        int icecreamTotalWeight = icecream * 2;
+        System.out.println(icecreamTotalWeight +" грамм");
+        int eggTotalWeight = egg * 4;
+        System.out.println(eggTotalWeight +" грамм");
+        int totalWeightOfRecipe = babana1TotalWeight + milkTotalWeight + icecreamTotalWeight + eggTotalWeight;
+        System.out.println("Общий вес рецепта " + totalWeightOfRecipe + " грамм");
+        float totalWeightOfRecipeKilo = totalWeightOfRecipe / 1000f;
+        System.out.println("Общий вес рецепта " + totalWeightOfRecipeKilo + " кг" );
+
+    // Задача 4
+        int lostWeightKg = 7;
+        int lostWeightGr = lostWeightKg * 1000;
+        System.out.println("Вес, который нужно сбросить " + lostWeightGr + "грамм");
+        int dailyLoss1 = 250;
+        int dailyLoss2 = 500;
+        int weightLossRate1 = lostWeightGr / dailyLoss1;
+        System.out.println(lostWeightGr / dailyLoss1 + " дней потребуется, чтобы сбросить 7 кг");
+        int weightLossRate2 = lostWeightGr / dailyLoss2;
+        System.out.println(lostWeightGr / dailyLoss2 + " дней потребуется, чтобы сбросить 7 кг");
+
+
+
+    // Задача 5
+        int mashaOldSalary = 67760;
+        double mashaIncrease = 1.1;
+        double mashaNewSalary = mashaOldSalary * mashaIncrease;
+        System.out.println("Маша.Новая зарплата " + mashaOldSalary * mashaIncrease);
```

Comment:
следует использовать уже вычисленное значение
```suggestion
        System.out.println("Маша.Новая зарплата " + mashaNewSalary );
```